src/SFA.DAS.ApplyService.Data/Queries/OversightReviewQueries.cs

```diff
@@ -94,7 +96,9 @@ public async Task<CompletedOversightReviews> GetCompletedOversightReviews(string
                               FROM Apply apply
 	                      INNER JOIN Organisations org ON org.Id = apply.OrganisationId
                           INNER JOIN OversightReview r ON r.ApplicationId = apply.ApplicationId
+                          LEFT JOIN Appeal Appeal on apply.ApplicationId = Appeal.ApplicationId
 	                      WHERE apply.DeletedAt IS NULL
+                          AND Appeal.ApplicationId IS NULL
```

Comment:
Match to the comment above if doing changes

Reply:
changed to Appeal.Status